main/ui/src/main/java/org/cryptomator/ui/mainwindow/VaultDetailLockedController.java

```diff
@@ -41,6 +44,15 @@
 		}
 	}
 
+	public void initialize() {
+		mainWindow.addEventFilter(KeyEvent.KEY_PRESSED, keyEvent -> {
+			if( keyEvent.getCode() == KeyCode.ENTER && vault.get() != null && vault.get().isLocked()) {
+				this.unlock();
+				keyEvent.consume();
+			}
+		});
+	}
+
```

Comment:
This should be handled via [default buttons](https://openjfx.io/javadoc/16/javafx.controls/javafx/scene/control/Button.html)